test/e2e/Tests/Fixtures/FunctionAppProcess.cs

```diff
@@ -50,11 +50,17 @@ public async Task InitializeAsync()
             switch (this.testLanguage)
```

Comment:
What is this "Intentional bad logic" comment about? Isn't this already a switch the comment promises in the future?

Reply:
Ah yep need to remove this comment